packages/fx-core/src/core/error.ts

```diff
@@ -230,3 +230,11 @@ export class ObjectIsUndefinedError extends SystemError {
     super(new.target.name, `Object ${name} is undefined, which is not expected`, CoreSource);
   }
 }
+
+export function SolutionConfigError(): UserError {
+  return new UserError(CoreSource, "SolutionConfigError", "Load solution context failed.");
```

Comment:
constructor is : UserError(name, message, source)

Reply:
thanks!